Add discord banner to org readme (#1)
PaperMC/.github: profile/README.md

# PaperMC

PaperMC is a Minecraft Software organization focusing on improving the Minecraft ecosystem with faster and more secure software.

**Quickstart:**

> - Learn more on [PaperMC.io](https://papermc.io) or [get started](https://docs.papermc.io/paper/getting-started) now!
> - You can download Paper [here](https://papermc.io/downloads)

Reviewer: ```suggestion
- Learn more and find downloads on [papermc.io](https://papermc.io)
```
The get started guide isn't org related

Author: Are you sure that you dont want to link directly to the downloads page?

Reviewer: Yes

Author: Okay 😃

> - Support us by [sponsoring](https://papermc.io/sponsors) PaperMC

Reviewer: ```suggestion
- Support us by donating through [OpenCollective](https://opencollective.com/papermc)
```

> - If you feel like talking to us, or need help you can either [join our Forum](https://forums.papermc.io/), or directly our Discord server:

Reviewer: ```suggestion
- Join our community by [visiting our forums](https://forums.papermc.io/) or chatting on our Discord server:
```
